test/e2e/apimachinery/coverage.go

```diff
@@ -0,0 +1,389 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apimachinery
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	unstructuredv1 "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/generated"
+)
+
+const (
+	retryMax = 10
+)
+
+var (
+	patch, _ = json.Marshal(jsonpatch.Patch{})
+)
+
+type resourceMap map[resourceMeta]*resource
+
+type resourceMeta struct {
+	group   string
+	version string
+	name    string
+}
+
+type resource struct {
+	group        string
+	version      string
+	name         string
+	namespaced   bool
+	verbs        []string
+	subresources resourceMap
+}
+
+var _ = SIGDescribe("Coverage", func() {
```

Comment:
This belong into some test/e2e/coverage folder. It tests coverage of Kubernetes APIs, it's not about generic API features.

Reply:
This comment still applies.